### DISCUSSION TITLE
Tiled Diffusion for Multi-GPU

### DISCUSSION THREAD

**Discussion by SephirothFFKH** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#126] [Ideas]
Would it be possible to have a new feature for Tiled Diffusion to use Multi-GPU for generation by sending tiles to each available GPUs simultaneously to speed up image generation? Thank you!

**Comment by Kahsolt** [comment]
Possibly not, the torch framework initializing and model weights are managed by sd-webui, not this extension could cope with :(
